[DOCS] CLI Clean-up (#7904)
great-expectations/great_expectations: docs/docusaurus/docs/guides/validation/advanced/how_to_deploy_a_scheduled_checkpoint_with_cron.md

### 1. Verify Checkpoint suitability

> First, verify that your Checkpoint is runnable. This Python code can retrieve and run a Checkpoint:

Reviewer: ```suggestion
Run the following command to verify that your Checkpoint runs:
```
